buildSrc/src/main/java/com/uber/okbuck/core/manager/DependencyManager.java

```diff
@@ -515,17 +515,17 @@ private static void computeSha256IfAbsent(File file, HashMap<String, String> sha
     sha256Map.computeIfAbsent(key, k -> DependencyUtils.shaSum256(file));
   }
 
-  private static HashMap<String, String> initSha256Cache(Project rootProject) {
-    File projectMappingFile = rootProject.file(OkBuckGradlePlugin.OKBUCK_SHA256);
+  private static HashMap<String, String> initSha256Cache(Project rootProject, ExternalDependenciesExtension externalDependenciesExtension) {
+    File projectMappingFile = rootProject.file(externalDependenciesExtension.getSha256Cache());
     try {
       return FileUtil.readMapFromJsonFile(projectMappingFile);
     } catch (IOException e) {
       return new HashMap<>();
     }
   }
 
-  private static void persistSha256Cache(Project rootProject, HashMap<String, String> sha256Map) {
-    File projectMappingFile = rootProject.file(OkBuckGradlePlugin.OKBUCK_SHA256);
+  private static void persistSha256Cache(Project rootProject, HashMap<String, String> sha256Map, ExternalDependenciesExtension externalDependenciesExtension) {
+    File projectMappingFile = rootProject.file(externalDependenciesExtension.getSha256Cache());
```

Comment:
I think this might not work if the specified file is not inside the root project

Reply:
it works. I checked by modifying it in okbuck and verified it was moved outside the project root

Reply:
We need to use aboslute path